blog/2022-08-05-know-your-customer-workflow.md

```diff
@@ -0,0 +1,125 @@
+
+During the initial KYC workflow, banks are asked by the FATF to follow a [risk-based approach](https://www.msg-compliance.de/en/glossary#r). In order to do that, they gather extensive details about you, which they also use to tailor offers of additional services or stock investments that match your [risk appetite](https://csrc.nist.gov/glossary/term/risk_appetite). It’s all about knowing who *you* are as a person. These KYC processes also help reduce fraud once you are an established customer. For example, if the bank knows that you live in Germany, it could identify credit card activity in the USA as a fraud transaction within minutes after activity in Germany and block it.
+
+The KYC workflow of a bank or a financial service needs to cover many different steps, and the following is a simplified view of a workflow:
+
+![KYC workflow from Conductor](https://i.imgur.com/odi99wK.png)
+
+This example covers various use cases mentioned above but is focused on the initial customer-acceptance process. It starts with collecting details from the customer, including personal identifiable information (PII), residency information, and financial details. Afterward, the initial validations are completed—e.g., checking if the customer is on the bank’s blacklist or verifying with regulators if the first name or last name is on a [sanction list](https://www.creditsafe.com/gb/en/blog/compliance/sanctions-lists-what-do-i-need-to-do.html).
+
+Based on the feedback from these validations, the workflow decides if the customer is acceptable for the bank or not. If the customer is not acceptable, the workflow will end, and the customer onboarding process stops. If the customer is acceptable, the workflow will calculate a risk class and assign a customer class before sending a notification to the customer.
+
+In the next section, you will build this workflow using Netflix Conductor.
+
+## Implementing KYC Workflows Using Netflix Conductor
+Before you start to build the workflow, there are a few things you need to do so you can locally test and execute the examples that you will build. Being able to execute and test workflows locally is key to ensuring developer productivity. Later you will implement your workflow for productive use on a server or on a Software as a Service offering like [Orkes](https://play.orkes.io/newWorkflowDef](https://play.orkes.io/newWorkflowDef).
+
+These are the prerequisites that you will need for the rest of this article:
+
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) on your PC.
```

Comment:
This is a bag of hurts. we should just ask folks to use playground.

Reply:
moving forward - yes - great plan